scripts/lib/utility-data-overrides.ts

```diff
@@ -180,7 +180,15 @@ export const OVERRIDES = new Map<string | number, string>([
   [15263, 'Potomac Edison'],
 
   // ME
+  // Versant has two districts, Bangor Hydro and Maine Public. They have
```

Comment:
will this be confusing to users if they don't realize Bangor hydro/Maine Public and Versant are the same? (or maybe that's obvious if you're a customer?) I looked up Bangor hydro and Versant came up so those seem to be the same, but Maine Public Utilities does seem to be its own thing https://www.maine.gov/mpuc/home

Reply:
That website is the state PUC. If you go to https://www.versantpower.com, it pops up a modal to make you choose the Bangor Hydro or Maine Public district:

<img width="861" alt="Screenshot 2024-04-16 at 10 16 24 PM" src="https://github.com/rewiringamerica/api.rewiringamerica.org/assets/107014/e312c16d-9a0a-4a3a-9c3e-09f56437fbf5">

So it seems to me that customers would know the name Versant (although it is new within the last few years, that's consistently their customer-facing brand) and also which Versant district they're in.

Reply:
Also I inspected the HTML of this form and it does actually have a mapping of zips to districts, right there in the `<select>` tag, so we can use that if we ever need to.